tests/servers/tests.py

```diff
@@ -256,6 +256,25 @@ def test_keep_alive_on_connection_with_content_length(self):
         finally:
             conn.close()
 
+    def test_head_content_length_removed(self):
+        conn = HTTPConnection(
+            LiveServerViews.server_thread.host, LiveServerViews.server_thread.port
+        )
+
+        conn.request("HEAD", "/head_view/?content-length=0")
+        response = conn.getresponse()
+        self.assertIsNone(response.getheader("Content-Length"))
+        conn.close()
+
+    def test_head_content_length_set(self):
+        conn = HTTPConnection(
+            LiveServerViews.server_thread.host, LiveServerViews.server_thread.port
+        )
+        conn.request("HEAD", "/head_view/?content-length=11151978")
+        response = conn.getresponse()
+        self.assertEqual(response.getheader("Content-Length"), "11151978")
+        conn.close()
```

Comment:
Move tests to the `tests/servers/test_basehttp.py`.

Reply:
moved